tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27,py36,py37,py38,py39,pypy,pypy3
```

Comment:
For the record, I've only listed the impls that I have installed here and can therefore test.